crates/fda/src/main.rs

```diff
@@ -634,6 +636,62 @@ async fn pipeline(action: PipelineAction, client: Client) {
             endpoint_name,
             action,
         } => endpoint(name, endpoint_name.as_str(), action, client).await,
+        PipelineAction::HeapProfile {
+            name,
+            pprof,
+            output,
+        } => {
+            let response = client
+                .get_pipeline_heap_profile()
+                .pipeline_name(name)
+                .send()
+                .await
+                .map_err(handle_errors_fatal(
+                    client.baseurl.clone(),
+                    "Failed to set obtain heap profile",
+                    1,
+                ))
+                .unwrap();
+            let mut byte_stream = response.into_inner();
+            let mut buffer = Vec::new();
+            while let Some(chunk) = byte_stream.next().await {
+                match chunk {
+                    Ok(chunk) => buffer.extend_from_slice(&chunk),
+                    Err(e) => {
+                        eprintln!("ERROR: Unable to read server response: {}", e);
+                        std::process::exit(1);
+                    }
+                }
+            }
+
+            match output {
+                None => {
+                    let mut tempfile = tempfile().unwrap_or_else(|e| {
+                        eprintln!("ERROR: Failed to create temporary file: {e}");
+                        std::process::exit(1);
+                    });
+                    tempfile.write_all(&buffer).unwrap();
+
+                    let mut args = pprof.split_whitespace();
+                    let mut command =
+                        Command::new(args.next().expect("`pprof` command cannot be empty"));
```

Comment:
Can we do a check if pprof is installed first and abort with an error giving guidance on how to install it? (as is already done in the command help)